<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontEditDialog</class>
 <widget class="QDialog" name="FontEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="38,24,10">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="styleLabel">
       <property name="text">
        <string>Font style</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="familyLabel">
       <property name="text">
        <string>Font</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="styleEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="sizeEdit">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="familyList"/>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="styleList"/>
     </item>
     <item row="2" column="2">
      <widget class="QListWidget" name="sizeList"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="familyEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="effectsGroupBox">
         <property name="title">
          <string>Effects</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <item>
           <widget class="QCheckBox" name="strikeoutCheckBox">
            <property name="text">
             <string>Strikeout</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="underlineCheckBox">
            <property name="text">
             <string>Underline</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="writingSystemLabel">
         <property name="text">
          <string>Writing System</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="writingSystemComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="sampleGroupBox">
       <property name="title">
        <string>Sample</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLineEdit" name="sampleEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
